drivers/sensor/bh1749/bh1749_trigger.c

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * This driver is heavily inspired by the apds9960_trigger.c driver.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <device.h>
+#include <gpio.h>
+#include <i2c.h>
+#include <misc/util.h>
+#include <kernel.h>
+#include <sensor.h>
+#include "bh1749.h"
+
+extern struct bh1749_data bh1749_driver;
```

Comment:
What's this? Such variable is not defined.

Reply:
removed

Reply:
Huh. Fixed in next